sprint0/sprint0/Enemies/GoriyaBoomerang.cs

```diff
@@ -14,31 +14,159 @@ public class GoriyaBoomerang : ISprite
         private List<Rectangle> sources;
         private int currFrame;
         private readonly int totalFrames, repeatedFrames;
+        SpriteEffects h = SpriteEffects.FlipHorizontally;
+        SpriteEffects v = SpriteEffects.FlipVertically;
+        enum Direction { left, right, up, down, ne, se, sw, nw }
```

Comment:
would it make more sense for these to be all cardinal dirs then? n s e w ne se sw nw